worldengine/draw.py

```diff
@@ -477,3 +554,8 @@ def draw_ancientmap_on_file(world, filename, resize_factor=1,
                     draw_biome, draw_rivers, draw_mountains, draw_outer_land_border, 
                     verbose)
     img.complete()
+
+def draw_scatter_plot_on_file(world, filename):
+    img = ImagePixelSetter(512, 512, filename)
```

Comment:
These are plots, not maps. They allow you to see the relationship between temperature and rainfall so you can be sure that the values are too far out of bounds. As such there is no reason for the size of the plot to be related to the size of the map (and since I generate maps that are 2k and 4k I really don't want plots that large).

I could add arguments so that there's a completely separate X and Y value for the scatter plot but it seems like an excessive complication that would lead to occasional user errors. Not many, granted, but it would still probably cause more problems than it resolves.